test/integration/query-interface/changeColumn.test.js

```diff
@@ -215,8 +214,8 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
 
           const newForeignKeys = await this.queryInterface.getForeignKeyReferencesForTable('users');
           expect(firstForeignKeys.length).to.be.equal(newForeignKeys.length);
-          expect(firstForeignKeys[0].columnName).to.be.equal('level_id');
-          expect(firstForeignKeys[0].columnName).to.be.equal(newForeignKeys[0].columnName);
+          expect(firstForeignKeys[0].tableColumnNames).to.eql(['level_id']);
```

Comment:
`.eql` is not declared in chai's typings

```suggestion
          expect(firstForeignKeys[0].tableColumnNames).to.equal(['level_id']);
```